package.xml

```diff
@@ -10,9 +10,9 @@
         <email>sammyk@php.net</email>
         <active>yes</active>
     </lead>
-    <date>2018-12-14</date>
+    <date>2019-01-07</date>
     <version>
-        <release>0.8.1</release>
+        <release>0.9.0</release>
         <api>0.8.1</api>
```

Comment:
Shouldn't we also update the API version?

Reply:
Yeah, I didn't change this one since none of the existing public API's changed but we did at least add a new public method or two. So I'll go ahead and bump that as well. But looking at other extensions in PECL, they only up the `<api>` tag if the API has changed. I think it technically did in our case do I'll update it. :)